hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSet.java

```diff
@@ -84,7 +85,7 @@ public long get(Runtime runtime) {
                 new LongProbeFunction<Runtime>() {
                     @Override
                     public long get(Runtime runtime) {
-                        return runtime.availableProcessors();
```

Comment:
I'm not sure about this one. Maybe we should have two probes to log both values? Could help to see if the `RuntimeAvailableProcessors` was overridden.

Reply:
Yeah, I forgot to comment on that. 
Because the RuntimeAvailableProcessors should be overridden only in tests, where probes are typically not used, I believe we should output the actual number. WDYT?

Reply:
I just had in mind that users could configure a higher core count than the machine has, which might lead to issues. But it seems that you actually would have to call `RuntimeAvailableProcessors.override()` for that (thought it would be somewhere in the configuration or GroupProperty). So this scenario seems to be unlikely, a single probe should be sufficient.